### PR TITLE
[inner-1947&1952] in rwSplit, multiple statements need to split

### PR DESCRIPTION
**Pull request by wenyh1**
Reason:  
  BUG inner 1947、1952
Type:  
  BUG/Improve  
Influences：  
   fix xx

### CODE REVIEW

File: src/main/java/com/actiontech/dble/services/rwsplit/RWSplitHandler.java [hunk 1]
```diff
@@ -43,29 +44,27 @@ public class RWSplitHandler implements ResponseHandler, LoadDataResponseHandler,
...
+        if (!StringUtil.isEmpty(executeSql)) {
+            mysqlService.execute(rwSplitService, executeSql);
+        } else if (originPacket != null) {
             mysqlService.execute(rwSplitService, originPacket);
-        } else if (isHint) {
-            //remove comment sentences
-            mysqlService.execute(rwSplitService, rwSplitService.getExecuteSql());
         } else {
-            //ensure that the character set is consistent with the client
-            mysqlService.execute(rwSplitService, rwSplitService.getExecuteSqlBytes());
```
Comment (ylinzhu) [inline src/main/java/com/actiontech/dble/services/rwsplit/RWSplitHandler.java:68]: It is necessary to judge here for coding reasons before considering

**Comment by wenyh1** [comment]
Need to consider coding scenario

**Comment by wenyh1** [comment]
create new pr

**Comment by wenyh1** [comment]
create new pr

**Pull request closed without merging**